src/main/java/com/cloudbees/plugins/credentials/CredentialsTypeFilter.java

```diff
@@ -199,11 +198,7 @@ public boolean filter(CredentialsDescriptor type) {
          */
         @NonNull
         public List<String> getClassNames() {
-            return ExtensionList.lookup(CredentialsDescriptor.class)
-                    .stream()
-                    .map(type -> type.getClass().getName())
-                    .filter(classNames::contains)
-                    .collect(Collectors.toList());
+            return new ArrayList<>(classNames);
```

Comment:
I couldn't find any references to this method anywhere so I don't know how to test if this works or not, all tests in this plugin pass.

Reply:
I've checked and the functionality for includes and excludes still works:

![image](https://user-images.githubusercontent.com/21194782/135581128-ecba8693-cd21-44c9-b7e3-5ff22023a45f.png)

Reply:
💡 What you tested in your screenshot is the `CredentialsProviderFilter`. 
For `CredentialsTypeFilter` it's the Types > Add > Includes / Excludes.
Manually tested, it still works 👍 (at least, not discovered flaw) 

(the last one, Restrictions, is covered by CredentialsProviderTypeRestriction)